src/components/Article.jsx

```diff
@@ -1 +1,33 @@
 // export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  text,
+  date,
+}) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="box">
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={dateISO} data-cy="date">
+          {dateLocaleString}
+        </time>
+      </p>
+
+      <p data-cy="text">
+        {text}
+      </p>
+    </div>
+  );
+};
```

Comment:
propTypes ? just a question. i had a linter troubles without

Reply:
> propTypes ? just a question. i had a linter troubles without
hmm interesting, my linter doesnt say nothing , and 0 mentions about propTypes in checklist aswell